applications/asset_tracker_v2/src/cloud/cloud_codec/cloud_codec.h

```diff
@@ -283,44 +300,24 @@ typedef void (*cloud_codec_evt_handler_t)(const struct cloud_codec_evt *evt);
 int cloud_codec_init(struct cloud_data_cfg *cfg, cloud_codec_evt_handler_t event_handler);
 
 /**
- * @brief Encode cloud codec neighbor cells data.
- *
- * @note LwM2M builds: This function does not output a list of objects, unlike other
- *		       functions in this API. The object references that are required to update
- *		       neighbor cell measurements are kept internal in the LwM2M utils library.
- *
- * @param[out] output String buffer for encoding result.
- * @param[in] neighbor_cells Neighbor cells data.
- *
- * @retval 0 on success.
- * @retval -ENODATA if data object is not marked valid.
- * @retval -ENOMEM if codec couldn't allocate memory.
- * @retval -EINVAL if the data is invalid.
- * @retval -ENOTSUP if the function is not supported by the encoding backend.
- */
-int cloud_codec_encode_neighbor_cells(struct cloud_codec_data *output,
-				      struct cloud_data_neighbor_cells *neighbor_cells);
-
-#if defined(CONFIG_LOCATION_METHOD_WIFI)
-/**
- * @brief Encode cloud codec Wi-Fi access points data.
+ * @brief Encode cloud codec cloud location data.
  *
  * @note LwM2M builds: This function does not output a list of objects, unlike other
  *		       functions in this API. The object references that are required to update
  *		       Wi-Fi access points are kept internal in the LwM2M utils library.
  *
  * @param[out] output String buffer for encoding result.
- * @param[in] wifi_access_points Wi-Fi access points.
+ * @param[in] cloud_location Cloud location data.
  *
  * @retval 0 on success.
  * @retval -ENODATA if data object is not marked valid.
  * @retval -ENOMEM if codec could not allocate memory.
  * @retval -EINVAL if the data is invalid.
  * @retval -ENOTSUP if the function is not supported by the encoding backend.
  */
-int cloud_codec_encode_wifi_access_points(struct cloud_codec_data *output,
-					  struct cloud_data_wifi_access_points *wifi_access_points);
-#endif
+int cloud_codec_encode_cloud_location(
+	struct cloud_codec_data *output,
+	struct cloud_data_cloud_location *cloud_location);
```

Comment:
I'm a little confused by the wording cloud location. That doesn't tell me much to be honest. Is there some common name for both ncell and wifi access point based location that can be used here instead?

Reply:
There is no common industry standard name for this. I've used `cloud location` term for methods that scan for technology specific cells/base stations/access point etc. and then sends those to cloud which does the location resolution. BT would be one more of these and there are some others that might pop up in the future.

We could think of GNSS as being device location type of method instead of cloud location because it scans for something and calculates the location within the device. And we could have some more this kind of methods in the future.

We have nrf_cloud_location library for this same purpose that handles nRF Cloud part of the cloud location.
Cloud side decided to use word `ground fix` in the cloud side for this. After it was in too many places I brought the name up and we concluded it would be removed. And then few days later they said it would be too much effort. So "ground fix" is used in nRF cloud side but this is not an industry standard terminology but invented by Nordic.

So let me know what the good name is :)